### PR TITLE
Fix layout

### PR DESCRIPTION
**Pull request by Vlad-coder-678**
https://trello.com/c/MoaB6rWn/28-fixlayout

### CODE REVIEW

**Review by Evgenia-cyber — APPROVED** [review]

File: poker/src/components/CardIssue/CardIssue.module.scss [hunk 1]
```diff
@@ -19,23 +19,34 @@
...
+  font-size: 1.2rem;
+  line-height: 4rem;
+  margin: 0;
   font-weight: 600;
 }
 
 .cardIssue_item p {
   position: absolute;
-  bottom: 0;
+  top: -9px;
```
Comment (AnAtoliy-AA) [inline poker/src/components/CardIssue/CardIssue.module.scss:30]: is it necessary to use negative values here&

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx [hunk 2]
```diff
@@ -25,79 +26,85 @@ interface Props {
...
 
-  useEffect(() => {
-    if (personalData.firstName === '') {
-      const { firstName } = personalData;
-      setError((state) => ({ ...state, firstName }));
-    }
-  }, [personalData]);
+  const room = isAdmin ? `room${socket.id}` : url;
+
+  console.log('SocketId', socket.id);
```
Comment (AnAtoliy-AA) [inline poker/src/components/ConnectToLobby/ConnectToLobby.tsx:43]: remove this string

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx [hunk 3]
```diff
@@ -25,79 +26,85 @@ interface Props {
...
-      }
-    };
-
-    socket.emit('login', { user, room }, callback);
+
+    if (personalData.firstName.length > 0) {
+      console.log('send form');
+      const { firstName, lastName, jobPosition } = personalData;
+      console.log('room:', room);
+      console.log('url', url);
```
Comment (AnAtoliy-AA) [inline poker/src/components/ConnectToLobby/ConnectToLobby.tsx:52]: a lot of logs)

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx [hunk 4]
```diff
@@ -25,79 +26,85 @@ interface Props {
...
+        role: userRole,
+      };
+
+      const callback = (response: ResponseFromSocket): void => {
+        console.log(response);
+
+        const { eventName, code, error: responseError, data } = response;
+
+        if (responseError) console.log(`${eventName}: ${code}: ${responseError}`);
+        else {
```
Comment (AnAtoliy-AA) [inline poker/src/components/ConnectToLobby/ConnectToLobby.tsx:72]: nesting condition
hard to read

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx [hunk 5]
```diff
@@ -25,79 +26,85 @@ interface Props {
...
+    } else {
+      setError('Please fill the form');
+      setIsError(true);
+    }
+  };
+
+  const handleNameChange = (e: ChangeEvent<HTMLInputElement>): void => {
+    const v = e.target.value;
+    const len = v.length;
+    if (len < 3) {
```
Comment (AnAtoliy-AA) [inline poker/src/components/ConnectToLobby/ConnectToLobby.tsx:97]: magic number

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx [hunk 6]
```diff
@@ -25,79 +26,85 @@ interface Props {
...
+      setError('*the field "Name" must be at least 3 characters');
+      setIsError(true);
+    } else if (len > 12) {
```
Comment (AnAtoliy-AA) [inline poker/src/components/ConnectToLobby/ConnectToLobby.tsx:100]: the same as before

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx [hunk 7]
```diff
@@ -25,79 +26,85 @@ interface Props {
...
+      setError('*the field "Name" must be no more than 12 characters');
+      setIsError(true);
+    } else {
+      setError('');
+      setIsError(false);
```
Comment (AnAtoliy-AA) [inline poker/src/components/ConnectToLobby/ConnectToLobby.tsx:105]: maybe it's better to unite this with upper string?

File: poker/src/components/LobbySetting/LobbySetting.tsx [hunk 8]
```diff
@@ -73,14 +73,14 @@ const LobbySetting: FC = () => {
...
         <Checkbox name={'master/player'} isChecked={settings.masterIsPlayer} onChange={handleChangeMasterOrPlayer} />
       </div>
-      <div className={styles.lobbyPage_setting_item}>
+      {/* <div className={styles.lobbyPage_setting_item}>
         <p>Changing card in round end:</p>
         <Checkbox
           name={'isChangeCardInRoundEnd'}
           isChecked={settings.isChangeCard}
           onChange={handleChangeCardInRoundEnd}
         />
```
Comment (AnAtoliy-AA) [inline poker/src/components/LobbySetting/LobbySetting.tsx:82]: a lot of comments in code

File: poker/src/pages/MainPage/MainPage.tsx [hunk 9]
```diff
@@ -25,9 +24,14 @@ const MainPage: FC = () => {
   };
 
   const handleEnterUrlGame = (e: ChangeEvent<HTMLInputElement>): void => {
-    setUrl(e.target.value);
+    setRoom(e.target.value);
   };
 
+  useEffect(() => {
+    const roomId = window.location.href.split('#')[1] ?? '';
```
Comment (AnAtoliy-AA) [inline poker/src/pages/MainPage/MainPage.tsx:31]: empty string

File: poker/src/components/CardIssue/CardIssue.module.scss (hunk 1 above)
Comment (Vlad-coder-678) [reply poker/src/components/CardIssue/CardIssue.module.scss:30]: this text can be moved with 'transform: translateY(-9px)', but this is an extra line of code

File: poker/src/pages/MainPage/MainPage.tsx (hunk 9 above)
Comment (AnAtoliy-AA) [reply poker/src/pages/MainPage/MainPage.tsx:31]: is it necessary to use ?? here?
may be || ?

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx (hunk 2 above)
Comment (Vlad-coder-678) [reply poker/src/components/ConnectToLobby/ConnectToLobby.tsx:43]: which is empty or with socketId?

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx (hunk 3 above)
Comment (Vlad-coder-678) [reply poker/src/components/ConnectToLobby/ConnectToLobby.tsx:52]: this is necessary at the development stage, we will remove it for release

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx (hunk 4 above)
Comment (Vlad-coder-678) [reply poker/src/components/ConnectToLobby/ConnectToLobby.tsx:72]: o yes. I myself do not understand what is written here))

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx (hunk 7 above)
Comment (Vlad-coder-678) [reply poker/src/components/ConnectToLobby/ConnectToLobby.tsx:105]: can you show what you mean?

File: poker/src/components/LobbySetting/LobbySetting.tsx (hunk 8 above)
Comment (Vlad-coder-678) [reply poker/src/components/LobbySetting/LobbySetting.tsx:82]: according to the task, the cards should be turned over after all participants have voted. if we keep up with the deadlines, then I will uncomment this code and use it. if we don't have time, I'll just delete it

**Review by GoodFellow23 — APPROVED** [review]

**Pull request merged**